anycast/.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+7.4.0
```

Comment:
can you please regenerate for 7.0.1 since that is the version used for all the other packages here. Would keep it consistent. We do want to update it later with a newer version, but we will do it as a separate effort.

Reply:
done